packages/jest-runner/src/runTest.ts

```diff
@@ -35,13 +36,13 @@ type RunTestInternalResult = {
 };
 
 function freezeConsole(
-  // @ts-ignore: Correct types when `jest-util` is ESM
-  testConsole: BufferedConsole | Console | NullConsole,
+  testConsole: BufferedConsole | CustomConsole | NullConsole,
   config: Config.ProjectConfig,
 ) {
+  // @ts-ignore: `_log` is `private` - we should figure out some proper API here
   testConsole._log = function fakeConsolePush(
```

Comment:
sounds like a nice followup to have a setter for that

Reply:
yeah, some sort of "disable logging" would be nice. We inject the instance into the sandbox though, so probably hide it behind a `Symbol` or something